src/Console/Command/FixCommand.php

```diff
@@ -506,6 +434,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @param string          $format
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return ReportInterface
+     */
+    protected function createReport($format, InputInterface $input, OutputInterface $output)
+    {
+        $reportClass = self::$reports[$format];
+
+        /** @var ReportInterface $report */
+        $report = new $reportClass();
+        $report->setDecoratedOutput($output->isDecorated());
```

Comment:
`DecoratedTextReport` and `TextReport` will different in string templates only.

May be I should make AbstractTextReport? Or should I remove `final` from `TextReport` and make `DecoratedTextReport` extends `TextReport` ?